builder/layouts/new.hbs

```diff
@@ -1,27 +1,20 @@
-{{#> head }}
-  <title>Add new snippet</title>
-  <style>
-    .layout {
-      display: flex;
-      flex-direction: column;
-    }
+{{#> head}}
+    <title>New Snippet - {{site.title}}</title>
+    <style>
+        body {
```

Comment:
if we insert some additional code in the `body` snippet this code will be broken. the previous solution under some conditions as well :(

Reply:
True, lol, we need screenshot diffing